### DISCUSSION TITLE
I used a template from the gallery for a website. I did not change the font specs from the template, why is the font slighly different?

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#8045] [Q&A]
### Description

This is what I used as template: 
![test12](https://github.com/quarto-dev/quarto-cli/assets/39966711/5a464c86-e9d3-4dce-8b32-d988d957a735)

But this is what I get on my website: 
![test11](https://github.com/quarto-dev/quarto-cli/assets/39966711/a67a366e-c0be-49f5-9fe0-93f0970c0fa3)

Can anyone help me understand why this difference occurs, even though I did not change anything related to the font? Thank you for your time.

**Comment by mcanouil** [comment]
I'm afraid asking to compare screenshots to get answers won't get you anywhere.

You will only get the same results if you are in the exact same setup, meaning exact same Quarto version and the exact same font availability and CSS/SCSS styling.
Quarto is using Bootstrap and updating it from time to time for instance.

Note that styling is heavily opinionated so no matter the reason why you wanted to know what's the difference beside the obvious difference of version, nothing is likely to be done here.

**Reply by ghost** [reply]
Thank you. Screenshots were just to give a visual example. The reason why I wanted to know is to understand why this difference occurred. I understand there may be a difference in Quarto versions; I don't understand the rest.

**Comment by cderv** [comment]
Font can be different if you don't have the same font available, or you can't access an online font. 

The website you looked as example  is using Source Sans Pro
![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/33a418de-2d83-46f8-bbf6-32fba48501ea)

but there are fallbacks as you can see. 

Look at the Rendered Font pane on your browser to see if you get the same fonts 

They are using Cosmo theme, and I believe these are the defaults fonts 
https://github.com/Openscapes/quarto-website-tutorial/blob/8da986e3263f5b862836d5f121a5afb6a1238c80/_quarto.yml#L49-L51

You need to inspect the HTML and CSS to understand the difference. If you have the exact same setup, it should be the same font
